xtask/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "xtask"
+version.workspace = true
+authors.workspace = true
+description.workspace = true
+homepage.workspace = true
```

Comment:
Can you change the description of this crate, please? Using something different from the Meilisearch description.

Reply:
Done!